File: src/main/java/itmo/is/project/mapper/resource/ResourceMapper.java
package itmo.is.project.mapper.resource;

import itmo.is.project.dto.resource.ResourceDto;
import itmo.is.project.mapper.EntityMapper;
import itmo.is.project.model.resource.Resource;
import org.mapstruct.Mapper;


@Mapper
public interface ResourceMapper extends EntityMapper<ResourceDto, Resource> {
}

File: src/main/kotlin/itmo/isproject/shared/resource/mapper/ResourceMapper.kt
package itmo.isproject.shared.resource.mapper

import itmo.isproject.shared.common.mapper.EntityMapper
import itmo.isproject.shared.resource.dto.ResourceDto
import itmo.isproject.shared.resource.model.Resource
import org.mapstruct.Mapper

@Mapper
interface ResourceMapper : EntityMapper<ResourceDto, Resource>